src/content/pages/thank-you.mdx

```diff
@@ -0,0 +1,354 @@
+This is the largest team, divided into several smaller teams. You can meet them onsite or when contacting our helpdesk, staffed weekly by rotating team members.
+
+Operations Team:
+
+- Aneta Popelová
+- Anežka Müller
+- Aris Nivorlis
+- George Margaritis
+- George Zisopoulos
+- Jakub Červinka
+- Jake Balaš
+- John Robert
+- Kateřina Vaňková
+- Liberta Gani
+- Mahe Iram Khan
+- Martin Borus
+- Moisés Guimarães
+- Niklas Mertsch
+- Piotr Gnus
+- Saurav Pandey
+- Simon Peter Dagbui
+
+
+## Discord Team
+
+If you have a valid ticket, you have received access to our Discord server where you can chat with fellow participants, speakers, or organisers. This team manages the Discord server, ensuring only participants have access and proper permissions.
+
+Huge thanks to **Niklas Mertsch** for leading Discord and bot management!
+
+The work on maintaining and updating the Discord bot can be found on the [EuroPython Discord bot repo](https://github.com/EuroPython/discord).
+
+
+## A/V Team
+Although an external supplier handles streaming and recording, this team communicates with the venue, manages digital screens, and handles unforeseen technical issues.
+
+Big thanks to **Raquel Dou**,  **Piotr Gnus** and **Anežka Müller** for A/V coordination.
+
+
+## Onsite Volunteers Team
+
+Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials.
+
+- xxx
+
+
+# Events & Workshops Organisers
+Beyond talks and tutorials, EuroPython includes engaging events and workshops.
+
+## Django Girls
+A workshop designed to inspire less represented groups, particularly women, to get interested in technology. Big thanks to Doreen Peace Nangira Wanyama for organizing!
+
+
+## Humble Data
+
+Data workshop designed for total beginners coming from less represented groups. Organisers and mentors:
+
+- Cheuk Ting Ho
+- Jodie Burchell
+- Martin Borus
+- Sena Şahin
+
+Special thanks to **Jodie Burchell** for coordinating beginner workshops.
+
+## Unconference Day
+In parallel with sprints and workshops, there was an unconference day featuring Q&A, discussions, and unconference-style sessions, inspired by the same concept hosted at PyCon CZ 23. Huge thanks to organisers **Daria Grudzien Linhart** and **Honza Javorek**.
+
+## PyLadies Events
+On Thursday, PyLadies hosted multiple events for PyLadies and allies to get together. Shoutout to **Maria José Molina** for coordination.
+## Community booths
+
+During the three main conference days, a few communities participated with a booth at the venue to tell people more about their communities and conferences. Thank you to **Maria José Molina** for coordination, and the volunteers that helped.
+## Sprints
+If you came to sprints, you could have seen open-source maintainers and people submitting PRs to projects. Many thanks to **Rodrigo Girão Serrão** for organising sprints and supporting all maintainers and sprint organisers.
```

Comment:
Feels unfair to get this shoutout when Anežka did everything. I just smiled and waved.